cmd/transform.go

```diff
@@ -35,21 +39,55 @@ var transformCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		pg, err := db.NewPostgreSQL(u, postgresSchema, nil)
-		if err != nil {
-			return err
-		}
-		defer pg.Close()
 
-		if cleanUp {
-			if err := pg.DropTable(); err != nil {
+		if strings.Contains(os.Getenv("DATABASE_URL"), "mongodb") {
+			mdb, err := db.NewMongoDB(mongoDatabase)
+			if err != nil {
 				return err
 			}
-			if err := pg.CreateTable(); err != nil {
+
+			if cleanUp {
+
+				err = mdb.DropCollection()
+				if err != nil {
+					return err
+				}
+
+				err = mdb.CreateCollection()
+				if err != nil {
+					return err
+				}
+
+				err = mdb.CreateIndexes()
+				if err != nil {
+					return err
+				}
+			}
+
```

Comment:
Podemos ser mais enxutos quanto às linhas em branco?
Aqui achei meio exagerado : /

```suggestion
			if cleanUp {
				err = mdb.DropCollection()
				if err != nil {
					return err
				}
				err = mdb.CreateCollection()
				if err != nil {
					return err
				}
				err = mdb.CreateIndexes()
				if err != nil {
					return err
				}
			}
```